### PR TITLE
Feature/enable target property access link fc

### PR DESCRIPTION
**Pull request by devtoni**

### CODE REVIEW

**Review by midudev — APPROVED** [review]

**Review by naxhh — APPROVED** [review]

File: components/dropdown/basic/src/index.js [hunk 1]
```diff
@@ -175,8 +178,8 @@ DropdownBasic.propTypes = {
 
 DropdownBasic.defaultProps = {
   expandOnMouseOver: false,
-  linkFactory: ({ href, className, children, title }) =>
-    <a href={href} className={className} title={title}>{children}</a>
+  linkFactory: ({ href, className, children, title, target }) =>
+    <a href={href} className={className} target={target} title={title}>{children}</a>
```
Comment (kikoruiz) [inline components/dropdown/basic/src/index.js:182]: What about using the `...rest` operator?

Comment (devtoni) [reply components/dropdown/basic/src/index.js:182]: I think it is better to restrict props we passed through but it would be ok too

Comment (kikoruiz) [reply components/dropdown/basic/src/index.js:182]:
```
linkFactory: ({ children, ...props }) =>
    <a {...props}>{children}</a>
```

**Review by davidmartin84 — APPROVED** [review]

**Review by siscu — APPROVED** [review]

File: components/dropdown/basic/src/index.js (hunk 1 above)
Comment (kikoruiz) [reply components/dropdown/basic/src/index.js:182]: In this case, we restrict the props we pass as per defined shape of links: https://github.com/SUI-Components/sui-components/blob/21b2a70705f790207b71677d4f66f20aafddc183/components/dropdown/basic/src/index.js#L154

Comment (kikoruiz) [reply components/dropdown/basic/src/index.js:182]: The thing is to write as less code as we can if it's possible ;)

**Review by kikoruiz — APPROVED** [review]

**Review by JoanClaret — APPROVED** [review]

**Review by davidbarna — APPROVED** [review]

**Pull request merged**